app/code/Magento/MediaGallery/Model/Keyword.php

```diff
@@ -18,7 +18,6 @@
 class Keyword extends AbstractExtensibleModel implements KeywordInterface
 {
     private const ID = 'id';
-
     private const KEYWORD = 'keyword';
```

Comment:
Remove extends AbstractExtensibleModel.
Constructor and getters can be generated by phpshtorm

Reply:
Why?